advocacy_docs/partner_docs/ImpervaDataSecurityFabric/04-ConfiguringImpervaDataSecurityFabric.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuring Imperva Data Security Fabric Agent'
+description: 'Walkthrough on configuring Imperva Data Security Fabric Agent'
+---
+
+Implementing the Imperva Data Security Fabric Agent, with EDB Postgres Advanced Server and PostgreSQL requires the following components:
+- The Imperva Agent installed on the same server as your EDB Postgres Advanced Server or PostgreSQL server.   
+- Database Security Gateway configured.
+- Management server configured.
+
+## Prerequisites
+
+- Install and configure the Imperva Data Security Fabric Management environment.
+- Register the Gateway to the Management server.
+- Install the Imperva Agent by running the installation file.
+- Register the Imperva Agent to the Gateway.
+
+## Configure Imperva Agent for EDB Postgres Advanced Server 
+The following steps show how to configure the Imperva Data Security Fabric Agent so it can be used to help monitor your EDB Postgres Advanced Server database or your PostgreSQL database.
+
+## Basic Imperva Data Security Fabric Configuration Overview
+
+The primary objective of the basic configuration is to identify what traffic you want to protect. This is accomplished by defining the basic building blocks that comprise the Imperva Data Security Fabric domain using a number of parameters including IP addresses, ports, protocols and more. The screen shot below illustrates basic configuration of these components.
+
+   ![ImpervaSitesTree](Images/ImpervaSitesTree.png)
+
+   ![ImpervaAgentConfiguration](Images/ImpervaAgentConfiguration.png)
+
+There are several configuration steps that can be taken with the Imperva Agent in your environment, as it is a highly configurable agent, to provide custom protection and support for your EDB Postgres Advanced Server and PostgreSQL databases.  This guide will list some basic configuration methods, but for a more complete list of configuration, please see the Basic Imperva Data Security Fabric Configuration Doc at: 
+https://docs.imperva.com/bundle/v14.9-database-activity-monitoring-user-guide/page/2383.htm
```

Comment:
Can we get rid of the link and instead make "Basic Imperva Data Security Fabric Configuration Doc" a hyperlink?